style/roads.mss

```diff
@@ -3397,6 +3401,25 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       }
     }
   }
+  [feature = 'highway_busway'] {
```

Comment:
This section is copied from the section below as is, i.e. setting it as `[feature = 'highway_busway'], [access = 'no'] {...}` wouldn't work.

Reply:
What I meant is: the whole section between lines 3405 and 3421 is copy pasted from 3432-3458, because just adding `[feature = 'highway_busway'],` before `[access = 'no']` on line 3423 did not work.

Reply:
But yeah, it doesn't make sense to have highway=road inside highway=busway rule, got you

Reply:
Fixed @HolgerJeromin, thanks